Scripts/1_Set_up_LER_folders.R

```diff
@@ -175,6 +219,21 @@ for(i in lakes){
                           key1 = "input", key2 = "light", key3 = "Kw",
                           key4 = "Simstrat", value = extinc_coef, verbose = F)
       
+      ### Light calibration
+      min_range = get_yaml_multiple(file = file.path(folder_root, folder_template_LER, "LakeEnsemblR.yaml"),

```

Comment:
I'm not sure if it is a good idea to have the scaling factors in the yaml file b.c. if they are overwritten or changed this could lead to problems. I would either define them in the Settings.R file or hardcode them here...